### DISCUSSION TITLE
How to silent eslint warnings on build?

### DISCUSSION THREAD

**Discussion by WilsonPhooYK** [vercel/next.js#26257] [Help]
![image](https://user-images.githubusercontent.com/83743898/122388336-602ac780-cfa2-11eb-96b5-091757b11d29.png)

These errors do lint on the files but did not show during build previously.
It appears now after ` yarn add --dev eslint-config-next` for NextJS v11.

Is there a --silent for these?

**Comment by housseindjirdeh** [comment]
Enabling `ignoreDuringBuilds` in the `eslint` property in your Next config will disable eslint running as a build step

https://nextjs.org/docs/api-reference/next.config.js/ignoring-eslint

**Reply by WilsonPhooYK** [reply]
Hi,
Thanks for the reply. But doing this will disable eslint all together. I still want it to check for errors but hide the warnings. It worked previously before I added `eslint-config-next`.

It seems like now I will have to set `"off"` for all these rules?

**Reply by housseindjirdeh** [reply]
Ah understood! 

> It seems like now I will have to set "off" for all these rules?

In the meantime, yes unfortunately. but I've just added support for the `--quiet` flag to `next lint` in this [PR](https://github.com/vercel/next.js/pull/26280). Once that lands, you'll be able to have `ignoreDuringBuilds` off and then run `next build && next lint` to only show errors and suppress warnings altogether.

I hope that's an acceptable workaround :). Please keep me posted if there's still something that can be improved. 🙏

**Reply by AlonMiz** [reply]
thanks,
i would rather have quiet config in next config. so I won't need to do the lining manually.

something like this
```
  eslint: {
    quiet: true,
  },
```

**Reply by harshsinghatz** [reply]
any follow up on this?

**Reply by u3ihtt** [reply]
is there any update?

**Reply by TheSwordBreaker** [reply]
anyany updates

**Reply by phendan** [reply]
This would be very useful, or alternatively an option to ignore specific rules or groups of rules. Running into trouble with eslint stylistic, errors for which are purely about code aesthetics and really shouldn't impact a build at all.

**Reply by ddri** [reply]
Echo the above. Feeling like overzealous linting is breaking deploys.

**Comment by northoperator** [comment]
hello, is there any solution for this in 2025? i cant make it shut up about warnings during nextjs build
